lib/markbind/lib/parser.js

```diff
@@ -358,20 +376,8 @@ Parser.prototype.includeFile = function (file, config) {
       decodeEntities: true,
     });
 
-    let actualFilePath = file;
-    if (!utils.fileExists(file)) {
```

Comment:
Probably should be added back so base/addressable pages can be properly resolved